### PR TITLE
Update monitoring stack

### PR DESCRIPTION
**Pull request by msimonin**
This is based on #244

### CODE REVIEW

File: enos/ansible/group_vars/all.yml [hunk 1]
```diff
@@ -51,3 +51,8 @@ patches:
     src: mariadb_bootstrap.yml
     dst: kolla/ansible/roles/mariadb/tasks/bootstrap.yml
     enabled: "no"
+
+# docker images
+grafana_docker_image: grafana/grafana:5.1.3
+influxdb_docker_image: influxdb:1.5.3 
+cadvisor_docker_image: google/cadvisor:v0.30.0
```
Comment (rcherrueau) [inline enos/ansible/group_vars/all.yml]: any reason for getting this defined here and in `roles/cadvisor/defaults/main.yml`?

File: enos/ansible/roles/influx/tasks/main.yml [hunk 2]
```diff
@@ -40,24 +46,20 @@
...
-  retries: 10
-  delay: 2
-
-- name: Create the annotation database
-  shell: "curl -s -XPOST http://{{ influx_vip }}:8086/query --data-urlencode 'q=CREATE DATABASE events'"
-  register: result
-  until: result.stdout == '{"results":[{}]}'
-  retries: 10
-  delay: 2
+- name: Create the telegraf database
```
Comment (rcherrueau) [inline enos/ansible/roles/influx/tasks/main.yml]: May I suggest: `name: Flush the annotation database (if exists)`

File: enos/ansible/group_vars/all.yml (hunk 1 above)
Comment (msimonin) [reply enos/ansible/group_vars/all.yml]: - Having those in `roles/*/defaults/main.yml` allows to have self-content roles
- Having those in `globals.yml` allow to centralize the current versions used in Enos.

File: enos/ansible/roles/influx/tasks/main.yml [hunk 3]
```diff
@@ -40,24 +46,20 @@
...
+  influxdb_database:
+    hostname: "localhost"
+    database_name: "events"
+    state: absent
 
+- name: Create the telegraf database
```
Comment (rcherrueau) [inline enos/ansible/roles/influx/tasks/main.yml]: May I suggest: `name: Create the annotation database`

File: enos/ansible/roles/influx/tasks/main.yml [hunk 4]
```diff
@@ -40,24 +46,20 @@
...
-- name: Create the cadvisor database
-  shell: "curl -s -XPOST http://{{ influx_vip }}:8086/query --data-urlencode 'q=CREATE DATABASE cadvisor'"
-  register: result
-  until: result.stdout == '{"results":[{}]}'
-  retries: 10
-  delay: 2
+- name: Create the telegraf database
+  influxdb_database:
+    hostname: "localhost"
+    database_name: "telegraf"
```
Comment (rcherrueau) [inline enos/ansible/roles/influx/tasks/main.yml]: What happened to the cadvisor database ?

Comment (msimonin) [reply enos/ansible/roles/influx/tasks/main.yml]: oups :)

File: enos/ansible/roles/influx/tasks/main.yml (hunk 2 above)
Comment (msimonin) [reply enos/ansible/roles/influx/tasks/main.yml]: sure

**Comment by msimonin** [comment]
Thanks for the review I'll send new patches in a few.

File: enos/ansible/group_vars/all.yml (hunk 1 above)
Comment (rcherrueau) [reply enos/ansible/group_vars/all.yml]: > Having those in roles/*/defaults/main.yml allows to have self-content roles

Oh, with a view to reuse roles in other projects I guess. Got it.

**Comment by msimonin** [comment]
done.

File: enos/ansible/roles/influx/tasks/main.yml [hunk 5]
```diff
@@ -40,24 +46,21 @@
     delay: 2
     timeout: 120
 
-- name: Create the cadvisor database
-  shell: "curl -s -XPOST http://{{ influx_vip }}:8086/query --data-urlencode 'q=CREATE DATABASE cadvisor'"
-  register: result
-  until: result.stdout == '{"results":[{}]}'
-  retries: 10
-  delay: 2
+- name: Create the annotation database
```
Comment (rcherrueau) [inline enos/ansible/roles/influx/tasks/main.yml]: I am pretty sure the correct label is "Create the cadvisor database"

**Comment by msimonin** [comment]
Redone.

**Pull request merged**